queries/grammars/bus.grammar

```diff
@@ -243,13 +243,8 @@ QBusWord/fall →
     | 'tólfa:kvk'_et_gr/fall
 
 QBusNumber/fall →
-    QBusNounSingular/fall QBusNr? QBusNumberWord
+    QBusNounSingular/fall QBusNr? UHeilTala
+    # UHeilTala is a nonterminal from the util modules, which
+    # matches any cardinal number and places it in the list result.numbers
```

Comment:
And does it allow "eitt", like the old QBusNumberWord did?

Reply:
Yep!
Should I add a specifier though so it only matches numbers in `_nf_hk`?
It would then only match queries like `... leið (númer) eitt` or `... leið (númer) tvö` but not `... leið (númer) einn` or `leið (númer) þrír`.

Reply:
Decided to allow incorrect cases/genders for the bus numbers (the queries are understandable and nonambiguous).